lib/ace/ext/papeeria/tex_completer.coffee

```diff
@@ -149,12 +166,45 @@ define((require, exports, module) ->
       else
         callback(null, @cache)
 
+  ###
+  # Show popup if token.type at current pos is one of allowedTypes[i]
```

Comment:
remove irrelevant [i] from the comments. if token type at the current pos is one of the given array elements.